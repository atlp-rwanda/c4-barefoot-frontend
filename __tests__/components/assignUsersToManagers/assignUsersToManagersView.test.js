import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import AssignUsers from '../../../src/components/views/assignUsersToManagers';

describe('<AssignUsers />', () => {
    let wrapper;
    it( 'It should Redirect if not logged in', ()=>{
        localStorage.removeItem('barefootUserToken');
        wrapper= shallow(<AssignUsers />);
        const component = toJson(wrapper);
        expect(component).toMatchSnapshot();
        expect(wrapper.find('Redirect')).toBeTruthy();
    });

    it( 'It should forward if logged in', () => {
        localStorage.setItem('barefootUserToken', 'ok');
        wrapper= shallow(<AssignUsers />);
        const component = toJson(wrapper);
        expect(component).toMatchSnapshot();
    });
});
