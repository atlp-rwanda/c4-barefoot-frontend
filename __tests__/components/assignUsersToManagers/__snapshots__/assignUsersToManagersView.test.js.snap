// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AssignUsers /> It should Redirect if not logged in 1`] = `
<Redirect
  to={
    Object {
      "pathname": "/login",
    }
  }
/>
`;

exports[`<AssignUsers /> It should forward if logged in 1`] = `<Connect(AssignUsersToManagers) />`;
